fastify.js

```diff
@@ -622,6 +636,12 @@ function fastify (options) {
     this[kErrorHandler] = func.bind(this)
     return this
   }
+
+  function printRoutes (opts = {}) {
+    // includeHooks:true - shortcut to include all supported hooks exported by fastify.Hooks
+    opts.includeMeta = opts.includeHooks ? opts.includeMeta ? supportedHooks.concat(opts.includeMeta) : supportedHooks : opts.includeMeta
```

Comment:
It is a little bit hard to read this statement and it is changing the input user object too
We could improve the readability